crates/ruff_python_formatter/src/other/arguments.rs

```diff
@@ -103,7 +101,12 @@ impl FormatNodeRule<Arguments> for FormatArguments {
                 // )
                 // ```
                 // TODO(konstin): Doesn't work see wrongly formatted test
-                parenthesized("(", &group(&all_arguments), ")")
+                parenthesized_with_dangling_comments(
+                    "(",
+                    dangling_comments,
+                    &group(&all_arguments),
+                    ")"
+                )
```

Comment:
Nit: Can we change the builder to `parenthesized("(", &group(&all_arguments)), ")").with_dangling_comments(dangling_comments)` to be more in line with how other builders work?

Reply:
i'm thinking about `parenthesized` and `parenthesized_without_dangling_comments` since the latter should be the exception

Reply:
I will handle separately since it would be unrelated to thi schange.